redfish/power.go

```diff
@@ -179,6 +179,20 @@ func (power *Power) PowerSupplyReset(memberID string, resetType ResetType) error
 	return power.Post(power.powerSupplyResetTarget, t)
 }
 
+// GetRedundancySet return the []PowerSupply in the target redundancy group based on the memberID.
+func (power *Power) GetRedundancySet(memberID int) []PowerSupply {
+	var powerSupplies []PowerSupply
+	for _, psLink := range power.Redundancy[memberID].redundancySet {
```

Comment:
Would probably be good to make sure `memberID` is actually present before accessible `.redundancySet`.